App-service-environments-ILB-deployments RA - refresh - fix (#2)
mspnp/app-service-environments-ILB-deployments: prepare_jumpbox.md

> - Log in the VM again. Open the Microsoft Store, then search WSL, choose Ubuntu, and install it.

Reviewer: This instruction says "Microsoft Store" -- which will never be the case.  Can we update this?

- Follow [these instructions](https://docs.microsoft.com/windows/wsl/install-on-server) to install WSL on your jumpbox. Since you are using

>   Windows Server 2019 you will need to skip the Windows Store part (not available in WS 2019) and download ubuntu directly from [this link](https://docs.microsoft.com/windows/wsl/install-manual#downloading-distributions).

Reviewer: Since you've moved to Ubuntu 20 in the instructions below, can you make sure that this instruction also says Ubuntu 20. 

```suggestion
  Windows Server 2019, you will need to skip the Windows Store part and instead download Ubuntu 20.04 directly from [this link](https://docs.microsoft.com/windows/wsl/install-manual#downloading-distributions).
```

- Open WSL Ubuntu. When prompted, enter admin user and password. Then execute this command:

>   ```

Reviewer: ```suggestion
  ```bash
```

And on the two code blocks below this one.

Author: Done with my own commit

sudo apt install npm
sudo npm install -g bower
sudo apt install zip
wget -q https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
sudo dpkg -i packages-microsoft-prod.deb
sudo add-apt-repository universe
sudo apt-get update
sudo apt-get install apt-transport-https
sudo apt-get update
sudo apt-get install dotnet-sdk-3.1
sudo apt-get install dotnet-sdk-5.0
```

## Download source code

Clone the code from the git repository.

> ```

Reviewer: ```suggestion
```bash
```

Author: Done with my own commit
